demos/toolbar/waterfall-flexible-toolbar-custom-style.html

```diff
@@ -80,7 +60,7 @@
         </p>
       </div>
     </main>
-    <footer>
+    <footer class="demo-toolbar-floating-footer">
```

Comment:
`demo-toolbar__floating-footer` or `demo-toolbar__footer--floating` comment from below

Reply:
If it were a child of `demo-toolbar`, yes, absolutely! In this particular case, though, it's a sibling of `demo-toolbar`, so it gets its own separate block class.